src/time-picker/time-display.jsx

```diff
@@ -80,14 +76,19 @@ const TimeDisplay = React.createClass({
   },
 
   render() {
-    let {
+    const {
       selectedTime,
       mode,
       affix,
       ...other,
     } = this.props;
 
-    let styles = {
+    const {
+      prepareStyles,
+      timePicker,
+    } = this.state.muiTheme;
+
+    const styles = {
```

Comment:
also these. sorry for nitpicking :sweat_smile: So many of these styles are instantiated for no reason since they never change.